pkg/engine/customscripts.go

```diff
@@ -3,50 +3,90 @@
 
 package engine
 
-func GetKubernetesB64Provision() string {
-	return getBase64CustomScript(kubernetesCustomScript)
+func GetKubernetesB64CSEMain() string {
+	return getBase64EncodedGzippedCustomScript(kubernetesCSEMainScript)
 }
 
-func GetKubernetesB64ProvisionSource() string {
-	return getBase64CustomScript(kubernetesProvisionSourceScript)
+func GetKubernetesB64CSEHelpers() string {
+	return getBase64EncodedGzippedCustomScript(kubernetesCSEHelpersScript)
 }
 
 func GetKubernetesB64HealthMonitorScript() string {
-	return getBase64CustomScript(kubernetesHealthMonitorScript)
+	return getBase64EncodedGzippedCustomScript(kubernetesHealthMonitorScript)
 }
 
-func GetKubernetesB64Installs() string {
-	return getBase64CustomScript(kubernetesInstalls)
+func GetKubernetesB64KubeletMonitorSystemdTimer() string {
+	return getBase64EncodedGzippedCustomScript(kubernetesKubeletMonitorSystemdTimer)
 }
 
-func GetKubernetesB64Configs() string {
-	return getBase64CustomScript(kubernetesConfigurations)
+func GetKubernetesB64KubeletMonitorSystemdService() string {
```

Comment:
All these exported funcs should have comments, but I guess the existing ones didn't either. 🤷‍♂️